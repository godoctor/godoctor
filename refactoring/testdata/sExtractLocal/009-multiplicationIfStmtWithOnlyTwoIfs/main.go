package main

import "fmt"

func main() {
	a := 1
	b := 2
	c := 3
	if a < c {
		fmt.Println("")
	}
	if a*b < c { // <<<<< extractLocal,12,4,12,7,newVar,pass
		fmt.Println("")
	}
}
