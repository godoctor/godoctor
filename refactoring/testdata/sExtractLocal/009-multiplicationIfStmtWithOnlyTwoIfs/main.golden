package main

import "fmt"

func main() {
	a := 1
	b := 2
	c := 3
	if a < c {
		fmt.Println("")
	}
	newVar := a * b
	if newVar < c { // <<<<< extractLocal,12,4,12,7,newVar,pass
		fmt.Println("")
	}
}
